spec/acceptance/target_specs.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+resource 'Targets' do
+  let(:user) { create(:user) }
+  let(:topic) { create(:topic) }
+  let(:request) { { target: attributes_for(:target).merge({ topic_id: topic.id }) } }
+
+  route 'api/v1/targets', 'Target index' do
+    post 'Create' do
+      context 'when user is not logged' do
+        example 'Not logged user' do
+          do_request(request)
+
+          expect(status).to eq 401
+        end
+      end
+
+      context 'when user is logged' do
+        header 'access-token', :access_token_header
+        header 'client', :client_header
+        header 'uid', :uid_header
+
+        example 'Logged user' do
```

Comment:
We should be more clear with the example title here. The user has to be logged in, but what we are actually testing here is the target creation